doc/source/user_guide/groupby.rst

```diff
@@ -31,29 +31,29 @@ following:
     * Filling NAs within groups with a value derived from each group.
 
 * **Filtration**: discard some groups, according to a group-wise computation
-  that evaluates True or False. Some examples:
+  that evaluates as True or False. Some examples:
 
-    * Discard data that belongs to groups with only a few members.
+    * Discard data that belong to groups with only a few members.
     * Filter out data based on the group sum or mean.
 
 Many of these operations are defined on GroupBy objects. These operations are similar
-to the :ref:`aggregating API <basics.aggregate>`, :ref:`window API <window.overview>`,
-and :ref:`resample API <timeseries.aggregate>`.
+to those of the :ref:`aggregating API <basics.aggregate>`,
+:ref:`window API <window.overview>`, and :ref:`resample API <timeseries.aggregate>`.
 
 It is possible that a given operation does not fall into one of these categories or
 is some combination of them. In such a case, it may be possible to compute the
 operation using GroupBy's ``apply`` method. This method will examine the results of the
-apply step and try to return a sensibly combined result if it doesn't fit into either
-of the above two categories.
+splitting step and try to return a sensibly combined result if it doesn't fit into either
```

Comment:
I agree that is is better.

Reply:
I will leave it as it is. 
As a user, my feedback is that the sentence is very hard to understand.

Reply:
What do you think about

> This method will examine the results of the apply step and try to sensibly combine them into a single result if it doesn't fit into either of the above two categories.

Reply:
A lot clearer, thanks!